AjobK/Athena
# 217 admin role

closes #217 

 **What is it supposed to do**
I've refactored the seeder and added seeders for the permissions and roles and created a middleware for the backend.

 **How can we test your branch**
You can test the branch by running npm run reset, it should create 4 accounts with 4 different roles, 1 default attachment, all the defined permissions and roles and a profile for each account.

 **What browser have been checked**
